app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -95,35 +161,31 @@
     }
   }
 
-  .edit-link{
-    float: right;
-    margin-top: .9rem;
-  }
-
-  .overview{
-    margin: 1rem auto;
-  }
-
   .content {
     padding-bottom: 8rem;
   }
 
   dd.accordion-navigation > a{
-    border-top: rgb(148, 148, 148) 1px solid;
+    border-bottom: rgb(148, 148, 148) 3px solid;
     background-color: transparent;
-    transition: .5s background-color;
+    transition: .2s background-color;
 
     // angle icon
     i.fa {
-      margin-right: 8px;
+      opacity: 0;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets